spec/services/clean_up_base_users_spec.rb

```diff
@@ -7,19 +7,21 @@
 
   subject(:call) { described_class.call }
 
-  let!(:registered_user) { create(:base_user, :registered, :with_bookmarks, :with_preferences, bookmarks_count: 3) }
-  let!(:stale_users) { create_list(:base_user, 5, :with_bookmarks, :with_preferences, bookmarks_count: 5) }
+  # Reduced from 3 to 2 bookmarks per user to speed up test
```

Comment:
very small reduction, I don't believe it will make significant difference, but OK
Again I'd removed comments as they will puzzle future developers :)